daemon/graphdriver/driver.go

```diff
@@ -152,6 +153,20 @@ type Checker interface {
 	IsMounted(path string) bool
 }
 
+// LocalGetFunc is a function definition that is identical if the file system
+// exists locally on the host.
+type LocalGetFunc func(string, string) (string, error)
+
+// WrapLocalGetFunc wraps the old graphdriver Get() interface (LocalGetFunc)
+// with the current graphdriver.Get() interface
+func WrapLocalGetFunc(id, mountLabel string, f LocalGetFunc) (containerfs.ContainerFS, error) {
```

Comment:
It might make the change a little bigger, but I would prefer to see the graph drivers just use `containerfs.NewLocalContainerFS` and not this wrap method. This adds an extra method call and is kind of complex, just returning nil or `containerfs.NewLocalContainerFS` is simple enough.